acceptance/tests/service_stop_start.rb

```diff
@@ -7,8 +7,21 @@
 # On teardown, restore configuration file on each agent
 teardown do
   agents.each do |agent|
+    # Create a valid config file
+    create_remote_file(agent, pxp_agent_config_file(agent), pxp_config_json_using_puppet_certs(master, agent).to_s)
+
+    # Ensure pxp-agent service is running
+    # PUP-6702 - on Windows, the service may be in the Paused state
+    # 'puppet resource service pxp-agent' will not work as expected
+    # Workaround: stop/start it using net
+    if(windows?(agent)) then
+      on(agent, 'net stop pxp-agent && net start pxp-agent')
```

Comment:
Note that if the service is already in a stopped state , net stop will throw an error and will _not_ run the net start command.  It may make more sense to run `net stop` as one command and ignore errors and then run `net start` e.g.

```
on(agent, 'net stop ...', :fail_on_fail => false)
on(agent, 'net start ...')
```